src/app/settings.py

```diff
@@ -79,6 +79,14 @@
 
 WSGI_APPLICATION = "app.wsgi.application"
 
+# Django debug toolbar settings
+
+if ENVIRONMENT == "development":
```

Comment:
using `ENVIRONMENT` is fine for me, but wouldn't it be better to use `DJANGO_DEBUG`?

Reply:
I just let it like this cause the `django-debug-toolbar` is a dev dependency :/. I think we should discuss this topic to find the best approach possible.